etna/transforms/math/lags.py

```diff
@@ -44,6 +47,7 @@ def __init__(self, in_column: str, lags: Union[List[int], int], out_column: Opti
 
         self.in_column = in_column
         self.out_column = out_column
+        self.in_column_regressor: Optional[bool] = None
```

Comment:
All the output columns of this transform are regressors, such transforms are marked with `FutureMixin`